tests/actions/vstudio/vc2010/test_compile_settings.lua

```diff
@@ -36,6 +36,8 @@
 	<PrecompiledHeader>NotUsing</PrecompiledHeader>
 	<WarningLevel>Level3</WarningLevel>
 	<Optimization>Disabled</Optimization>
+	<ExceptionHandling>Sync</ExceptionHandling>
+	<RuntimeTypeInfo>true</RuntimeTypeInfo>

```

Comment:
And here is the crux of my issue, evidence that the number of explicitly defined things increases, regardless if the user actually defined them or not.
If the case is that premake has prescribed firm defaults for these API's, and VS's defaults contradict that prescription, then I may accept this as necessity.